### PR TITLE
chore: Docs: Cleanup Microsoft link

### PR DESCRIPTION
**Pull request by nschonni**
Ran the script from the docs-authoring-pack.
Looks like mostly helped remove locales

### CODE REVIEW

File: docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ If you examine the "Sent a message over a channel" trace, you may see the follow
        <TraceData>
           <DataItem>
              <TraceRecord xmlns="http://schemas.microsoft.com/2004/10/E2ETraceEvent/TraceRecord" Severity="Information">
-                 <TraceIdentifier>http://msdn.microsoft.com/library/System.ServiceModel.Channels.MessageSent.aspx</TraceIdentifier>
```
Comment (nschonni) [inline docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: Either URL is a 404, but I'm not sure if this is supposed to be HTTP like the schema URLs are

File: docs/fsharp/get-started/get-started-vscode.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the
 To begin, ensure that you have [F# and the Ionide plugin correctly installed](install-fsharp.md#install-f-with-visual-studio-code).
 
 > [!NOTE]
-> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **OSX**, a simpler way to get started is to use the [command line tools](https://docs.microsoft.com/en-us/dotnet/fsharp/get-started/get-started-command-line).
+> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **OSX**, a simpler way to get started is to use the [command line tools](https://docs.microsoft.com/dotnet/fsharp/get-started/get-started-command-line).
```
Comment (NextTurn) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **macOS**, a simpler way to get started is to use the [command line tools](get-started-command-line.md).
```

File: docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: not sure either. @Lxiamail can you help? should this perhaps link to https://docs.microsoft.com/dotnet/framework/wcf/diagnostics/tracing/system-servicemodel-channels-messagesent?

**Review by JRAlexander — APPROVED** [review]
LGTM for ML.NET related files. Thanks, @nschonni!

File: docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: Pinging @HongGit to see if she can help

Comment (nschonni) [reply docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: @mairaw I've just removed this to unblock the PR

**Review by BillWagner — APPROVED** [review]
This LGTM. I’ll :shipit: now.

**Pull request merged**